azure-pipelines.yml

```diff
@@ -1,78 +1,46 @@
-# Node.js with React
-# Build a Node.js project that uses React.
-# Add steps that analyze code, save build artifacts, deploy, and more:
-# https://docs.microsoft.com/azure/devops/pipelines/languages/javascript
-
-trigger:
-- master
-- develop
+trigger: none	 
```

Comment:
What does this pipeline do and when should it be used?

We're also missing the release pipeline which takes a zip file and publishes it to azure.

Reply:
This pipeline run after changes have been pushed to master. The release pipleline is available [here](https://dev.azure.com/CopticDevs/hazzat-api/_release).

We still need to add a release pipeline, but that is not part of this PR.